core-bundle/src/EventListener/PrettyErrorScreenListener.php

```diff
@@ -182,13 +182,24 @@ private function renderTemplate(string $template, int $statusCode, ExceptionEven
             return;
         }
 
+        // For BC reasons, check if a bundle template exists
```

Comment:
```suggestion
        // Backwards compatibility: check if a bundle template exists
```
So that we find this when we search for `"Backwards compatibility"` in Contao 6 ☺️